vividus-soft-assert/src/main/java/org/vividus/softassert/issue/KnownIssueIdentifier.java

```diff
@@ -30,6 +30,7 @@
     private Pattern assertionCompiledPattern;
     private boolean failTestCaseFast;
     private boolean failTestSuiteFast;
+    private String description;
```

Comment:
```suggestion
    private Optional<String> description;
```
?

Reply:
do you suggest to convert optional in setter? I'm not sure if is supports auto setting of non-specified in JSON fields to Optional type, i'll check